src/pages/workspaces/workspace/jobHistory/CallTable.js

```diff
@@ -94,7 +94,7 @@ const CallTable = ({ namespace, name, submissionId, workflowId, callName, callOb
           {
             size: { basis: 200, grow: 2 },
             field: 'callCachingResult',
-            headerRenderer: () => h(Sortable, { sort, field: 'callCachingResult', onSort: setSort }, ['Call Caching Result']),
+            headerRenderer: () => h(Sortable, { sort, field: 'callCachingResult', onSort: setSort }, ['Task Caching Result']),
```

Comment:
"Call caching" isn't the most intuitive name but it's something users are familiar with, we should leave this as-is.